volume-providers/softlayer/block/block_volume_manager_test.go

```diff
@@ -53,19 +53,19 @@ var _ = Describe("SLBlockSession", func() {
 		}
 	})
 
-	Describe("VolumeCreate", func() {
+	Describe("CreateVolume", func() {
 		var (
-			capacityItemPrices   []datatypes.Product_Item_Price
-			itemPrices           map[string]*datatypes.Product_Item_Price
+			//capacityItemPrices   []datatypes.Product_Item_Price
+			//itemPrices           map[string]*datatypes.Product_Item_Price
 			storagespacePrice    []datatypes.Product_Item_Price
-			itemPriceErrors      map[string]error
+			//itemPriceErrors      map[string]error
 			productPackages      []datatypes.Product_Package
 			networkStorages      []datatypes.Network_Storage
 			networkIscsiStorages []datatypes.Network_Storage_Iscsi
 		)
 
 		BeforeEach(func() {
-			capacityItemPrices = []datatypes.Product_Item_Price{
+			/*capacityItemPrices = []datatypes.Product_Item_Price{
```

Comment:
is it not needed anymore?

Reply:
This is the test data. we will check later actually I did not do any changes for this but tried to check `make test` hence got error thats why I commented